apps/chess-training-app/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM node:22-alpine AS builder
+WORKDIR /app
+
+COPY package.json yarn.lock ./
+COPY . .
+
+RUN yarn install --frozen-lockfile
+RUN yarn build
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)